insights/specs/default.py

```diff
@@ -339,6 +339,7 @@ class DefaultSpecs(Specs):
     lsblk = simple_command("/bin/lsblk")
     lsblk_pairs = simple_command("/bin/lsblk -P -o NAME,KNAME,MAJ:MIN,FSTYPE,MOUNTPOINT,LABEL,UUID,RA,RO,RM,MODEL,SIZE,STATE,OWNER,GROUP,MODE,ALIGNMENT,MIN-IO,OPT-IO,PHY-SEC,LOG-SEC,ROTA,SCHED,RQ-SIZE,TYPE,DISC-ALN,DISC-GRAN,DISC-MAX,DISC-ZERO")
     lscpu = simple_command("/usr/bin/lscpu")
+    lsinitrd_kdump_image = foreach_execute(kdump_initramfs_image.kdump_image, "/usr/bin/lsinitrd %s")
```

Comment:
@TZ3070 Instead of adding a new datasource, you can use the listdir spec_factory function to achieve the same result, example below.

`lsinitrd_kdump_image = foreach_execute(listdir("/boot/*kdump.img"), "/usr/bin/lsinitrd /boot/%s")`

Reply:
Update, thanks.